policyengine_us/variables/gov/states/mt/tax/income/exemptions/mt_interest_exemptions.py

```diff
@@ -0,0 +1,26 @@
+from policyengine_us.model_api import *
+
+
+class mt_interest_exemptions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Montana interest exemptions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://mtrevenue.gov/wp-content/uploads/dlm_uploads/2022/12/Form-2-2022-Instructions.pdf#page=25"
+    defined_for = StateCode.MT
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(period).gov.states.mt.tax.income.exemptions.interest
+        filing_status = tax_unit("filing_status", period)
+        cap = p.cap[filing_status]
+        person = tax_unit.members
+        interest_income = add(person, period, ["taxable_interest_income"])
```

Comment:
```suggestion
        interest_income = add(tax_unit, period, ["taxable_interest_income"])
```

Reply:
it is a person variable

Reply:
we are adding all the person level inputs into one tax unit hence the `add` function